protobuf/model_config.proto

```diff
@@ -1382,6 +1382,24 @@ message ModelSequenceBatching
     //@@     The optional field to specify the initial state for the model.
     //@@
     repeated InitialState initial_state = 5;
+
+    //@@  .. cpp:var:: bool use_single_buffer
+    //@@
+    //@@     The optional field to use a single buffer for both input and output state.
+    //@@     The default value is false.
+    //@@
+    bool use_single_buffer = 6;
```

Comment:
Can you add a few sentences on the performance impact of `use_single_buffer = true` if any? And what are the cases that user may want to pick one over the other?

Reply:
added